onix/couple/couple_openmc.py

```diff
@@ -568,7 +568,10 @@ def _set_statepoint(self, path = os.getcwd()):
     def _set_kinf(self):
 
         statepoint = self.statepoint
-        kinf = statepoint.k_combined
+        if ((int(openmc.__version__.split('.')[1]) == 13) and (int(openmc.__version__.split('.')[2]) >= 1)) or (int(openmc.__version__.split('.')[1]) > 13):
+            kinf = statepoint.keff
+        else:
+            kinf = statepoint.k_combined
```

Comment:
Its probably nicer to reuse the version check result from the density function:

```suggestion
        if old_openmc:
            kinf = statepoint.k_combined
        else:
            kinf = statepoint.keff
```

Reply:
I think your solution is good, although I dislike using global variables like that. An alternative option is to use a `try except` clause. I'm not sure if there are solid reasons to prefer one over the other.
```suggestion
        try:
            kinf = statepoint.keff
        except AttributeError:
            kinf = statepoint.k_combined
```
Feel free to commit whichever one you prefer.